### PR TITLE
Speak.repeat

### PR DESCRIPTION
**Pull request by blendaf**
Rebased on branch "adds create deck feature"

### CODE REVIEW

File: presentational/RepeatView.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+          <button onClick={submitFunction} type="button">
+            Create deck
+          </button>
+        </DeckBox>
+        <Modal modalVisible={modalState}>
+          <ModalText>Deckname can not be empty.</ModalText>
+          <button type="button" onClick={dismissModal}>
+            close
+          </button>
+        </Modal>
```
Comment (juliuscc) [inline presentational/RepeatView.js]: Wouldn't it be easier to disable the submit button when the input is an empty string?

File: presentational/RepeatView.js [hunk 2]
```diff
@@ -0,0 +1,218 @@
+import styled from 'styled-components'
+import { Search as SearchIcon } from 'styled-icons/zondicons/Search'
+import React from 'react'
+import Link from 'next/link'
+
+const RepeatWrapper = styled.div`
+  width: 100%;
+  height: 100vh;
```
Comment (juliuscc) [inline presentational/RepeatView.js]: What does this do?

File: presentational/RepeatView.js [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  background-color: ${({ theme }) => theme.colors.focusBackground};
+  border: none;
+  width: 100px;
+  height: 50px;
+  color: ${({ theme }) => theme.colors.black};
+  font-family: ${({ theme }) => theme.fonts.text};
+  font-size: 15px;
+  font-weight: 300;
+  cursor: pointer;
+`
```
Comment (juliuscc) [inline presentational/RepeatView.js]: These could be one I think

File: presentational/RepeatView.js (hunk 1 above)
Comment (blendaf) [reply presentational/RepeatView.js]: Fixed

File: presentational/RepeatView.js (hunk 2 above)
Comment (blendaf) [reply presentational/RepeatView.js]: fixed

File: presentational/RepeatView.js (hunk 3 above)
Comment (blendaf) [reply presentational/RepeatView.js]: Fixed

**Review by juliuscc — APPROVED** [review]

**Pull request merged**